mne_bids/commands/mne_bids_raw_to_bids.py

```diff
@@ -35,6 +36,9 @@ def run():
     parser.add_option('--run', dest='run',
                       help='The run number for this dataset.',
                       metavar='run')
+    parser.add_option('--acq', dest='acq',
+                      help='The acquisition parameter.',
+                      metavar='acq')
     parser.add_option('--kind', dest='kind',
```

Comment:
`kind` is still in the cli tool

Reply:
I didn't change how the CLI works. It should work just as before ...

Reply:
it's mostly a concern over unified api experience. after talking offline, I will wait til your next commit(s) to comment further

Reply:
yeah I see. I'm not sure how to expose a modular API for CLI though :) It's kind of similar for MNE report though. The Python API is really nice but the CLI is just a beast with many parameters. There are ways of chaining commands too but I think that would be longer discussion + bigger diff + more work etc. I would leave it out of the scope of this PR and release